Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/RoundImageEx.xaml

```diff
@@ -0,0 +1,125 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <Style TargetType="controls:RoundImageEx">
+        <Setter Property="Background" Value="Transparent" />
+        <Setter Property="Foreground" Value="{ThemeResource ApplicationForegroundThemeBrush}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:RoundImageEx">
+                    <Grid>
+                        <Ellipse x:Name="PlaceholderEllipse"
+                                 Width="{TemplateBinding Width}"
+                                 Height="{TemplateBinding Height}"
+                                 HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+                                 VerticalAlignment="{TemplateBinding VerticalAlignment}"
+                                 Opacity="1.0"
+                                 Stroke="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Stroke}"
+                                 StrokeThickness="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=StrokeThickness}">
```

Comment:
I'm not sure this is applicable. From what that SO says, this is because the OP needed to basically convert to another value. As long as Stroke and StrokeThickness on RoundImageEx are using the same property types as Ellipse, then no conversion should be needed and TemplateBinding can be used.